generators/pdf_to_text/pdf_to_text/config.py

```diff
@@ -0,0 +1,35 @@
+from util.configs import build_generator_function_config
+from util.enums import State, RefineryDataType, BricksVariableType, SelectionType
+from . import pdf_to_text, INPUT_EXAMPLE
+
+
+def get_config():
+    return build_generator_function_config(
+        function=pdf_to_text,
+        input_example=INPUT_EXAMPLE,
+        issue_id=329,
+        tabler_icon="file-type-pdf",
+        min_refinery_version="1.7.1",
```

Comment:
As this brick is not available in refinery, the min version is not really applicable. How do we want to tackle this? Maybe insert a placeholder value (like 0.0.0) here? Not sure.